includes/Admin/Settings_Screens/Advertise.php

```diff
@@ -287,7 +288,7 @@ private function try_render_experimental_view() {
 		$translate = function( $text ) {
 			return $this->get_escaped_translation( $text );
 		};
-		$translate_with_link = function( $hyperlink, $link_text, $pretext, $rest_of_text ) {
+		$translate_with_link = function( $hyperlink, $link_text, $pretext, $rest_of_text ) use(&$translate) {
 			return $translate( $pretext ) . " <a href='<?php echo " . $hyperlink . "?>' >" . $translate( $link_text ) . "</a>" . $translate( $rest_of_text );
 		};
```

Comment:
Why aren't these moved to separate methods? You could even add this to the [Util](https://github.com/woocommerce/facebook-for-woocommerce/blob/develop/includes/fbutils.php) class.

Reply:
Good point. I'll do that.